src/Dofs/ConstraintHandler.jl

```diff
@@ -67,14 +67,14 @@ struct ConstraintHandler{DH<:AbstractDofHandler,T}
     free_dofs::Vector{Int}
     inhomogeneities::Vector{T}
     dofmapping::Dict{Int,Int} # global dof -> index into dofs and inhomogeneities
-    bcvalues::Vector{BCValues{T}}
+    bcvalues::Vector{BCValues{Float64}} # BCValues are just the shape functions, not the BCs themselves
     dh::DH
     closed::ScalarWrapper{Bool}
 end
 
-function ConstraintHandler(dh::AbstractDofHandler)
+function ConstraintHandler(dh::AbstractDofHandler;bctype=Float64)
```

Comment:
An interface where you pass the type as the first argument:
```suggestion
function ConstraintHandler(T::Type{<:Number}, dh::AbstractDofHandler)
```
would be more typical I think.